bot/connector-iadvize/src/main/kotlin/IadvizeBuilder.kt

```diff
@@ -0,0 +1,118 @@
+
+/*
+ * Copyright (C) 2017/2022 e-voyageurs technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.tock.bot.connector.iadvize
+
+import ai.tock.bot.connector.ConnectorType
+import ai.tock.bot.connector.iadvize.model.response.conversation.QuickReply
+import ai.tock.bot.connector.iadvize.model.response.conversation.payload.TextPayload
+import ai.tock.bot.connector.iadvize.model.response.conversation.Duration
+import ai.tock.bot.connector.iadvize.model.response.conversation.reply.*
```

Comment:
avoid * imports

Reply:
ok (automatic change by intellij) it's fixed